src/python/grpcio_tests/tests/health_check/_health_servicer_test.py

```diff
@@ -22,21 +23,36 @@
 
 from tests.unit import test_common
 
+from six.moves import queue
+
+_QUEUE_TIMEOUT_S = 5
```

Comment:
nit: How about using the constants from `tests.unit.framework.common.test_constants`

Reply:
Done